Library/Formula/smpeg.rb

```diff
@@ -0,0 +1,46 @@
+class Smpeg < Formula
+  desc "SDL MPEG Player Library"
+  homepage "http://icculus.org/smpeg/"
+  url "svn://svn.icculus.org/smpeg/tags/release_0_4_5/"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "sdl"
+  depends_on "gtk+" => :optional
+
+  def install
+    args = %W[
+      --disable-debug
+      --disable-dependency-tracking
+      --prefix=#{prefix}
+      --disable-sdltest
+    ]
+
+    # For non-/usr/local installs
+    sdl = Formula["sdl"]
+    args << "--with-sdl-prefix=#{sdl.opt_prefix}"
```

Comment:
This can become `--with-sdl-prefix=#{Formula["sdl"].opt_prefix}` like the other formula.